packages/styling/src/styles/GeneralStyles.ts

```diff
@@ -7,4 +7,15 @@ export const normalize: IRawStyle = {
   margin: 0,
   padding: 0,
   boxSizing: 'border-box'
+};
+
+/**
+ * Prevents user selection of text elements.
+ * @param value The value of userSelect
+ */
+export const setUserSelect = (value: 'initial' | 'inherit' | 'unset' |
```

Comment:
Do we need a util function for setting a single property? This is overkill, since removing all the vendor prefixes now makes the SASS mixin unnecessary.

Reply:
OWA uses this mixin directly, and since mergeStyles takes care of the vendor prefixes I omitted them. I figured removing the function entirely would break contracts.
@MLoughry @micahgodbolt @battletoilet Can you chime in on whether this is still necessary?

Reply:
Yeah, we don't need to have this function here. Just make sure the userSelect is defined in IRawStyle with those possible values. OWA can continue to use the Sass mixin, and anything converted to getStyles will simply use the raw CSS property.

Reply:
Cool, I'll close the PR.